packages/application-config/test/fixtures/config-simple.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "avengers-app",
+  "entryPointUriPath": "avengers",
+  "cloudIdentifier": "gcp-eu",
+  "env": {
+    "production": {
+      "url": "https://avengers.app"
+    }
+  }
+}
```

Comment:
This is the simplest version that a Custom App can have. As you can see, most of the final values are inherited from this.